preflight_check/task.py

```diff
@@ -558,3 +564,94 @@ def wait(self):
         if not result:
             self.thread_stop_event.set()
             # should be an error here
+
+
+class TaskFixSBD(Task):
```

Comment:
Better to add __doc__ to describe what if this class for

Reply:
Done.